Lib/pathlib.py

```diff
@@ -115,8 +115,8 @@ class _WindowsFlavour(_Flavour):
     is_supported = (os.name == 'nt')
 
     drive_letters = (
-        set(chr(x) for x in range(ord('a'), ord('z') + 1)) |
-        set(chr(x) for x in range(ord('A'), ord('Z') + 1))
+        {chr(x) for x in range(ord('a'), ord('z') + 1)} |
```

Comment:
This should be simplified to set('abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ')
or if string is already imported to set(string.ascii_letters)